Code/Tools/ProjectManager/Source/ProjectsScreen.h

```diff
@@ -34,7 +34,7 @@ namespace O3DE::ProjectManager
 
     public:
         explicit ProjectsScreen(QWidget* parent = nullptr);
-        ~ProjectsScreen();
+        ~ProjectsScreen() = default;
```

Comment:
FYI you can default this in the cpp file using `ProjectsScreen::~ProjectScreen() = default`.

That has uses if the ProjectsScreen class definition declares a unique_ptr to an incomplete type.
In which case having the destructor in the header would force the class definition to have complete objects for it's pointer members at that point.

Now you don't have to move in the destructor back to the .cpp in this case.
I am just providing information in this comment.